store/CSDK/src/CarbonReader.cpp

```diff
@@ -19,15 +19,14 @@
 #include <jni.h>
 
 jobject CarbonReader::builder(JNIEnv *env, char *path, char *tableName) {
-
     jniEnv = env;
     jclass carbonReaderClass = env->FindClass("org/apache/carbondata/sdk/file/CarbonReader");
     jmethodID carbonReaderBuilderID = env->GetStaticMethodID(carbonReaderClass, "builder",
         "(Ljava/lang/String;Ljava/lang/String;)Lorg/apache/carbondata/sdk/file/CarbonReaderBuilder;");
-    jstring jpath = env->NewStringUTF(path);
+    jstring jPath = env->NewStringUTF(path);
     jstring jtableName = env->NewStringUTF(tableName);
     jvalue args[2];
-    args[0].l = jpath;
+    args[0].l = jPath;
     args[1].l = jtableName;
     carbonReaderBuilderObject = env->CallStaticObjectMethodA(carbonReaderClass, carbonReaderBuilderID, args);
     return carbonReaderBuilderObject;
```

Comment:
No need to return object because we are storing this variable. make it void.

Reply:
ok